worker_test.go

```diff
@@ -78,3 +80,25 @@ func TestWorkerImportHistogram(t *testing.T) {
 	wm := w.Flush()
 	assert.Len(t, wm.histograms, 1, "number of flushed histograms")
 }
+
+func TestWorkerImportCardinality(t *testing.T) {
+	w := NewWorker(1, nil, logrus.New())
```

Comment:
I think we can actually do `log` here instead of `logrus.New()`, because it will pick up the global one initialized in server.go

Reply:
ooh yes